kuka_kr270_support/package.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>kuka_kr270_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the KUKA KR 270 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for KUKA KR 270 manipulators. This currently includes the R2700 only.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>KR 270 R 2700 - Default</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information
+      found in the online datasheet of the Kuka KR 270 R2700 ultra.
+      <!-- <a href="https://www.kuka.com/-/media/kuka-downloads/imported/6b77eecacfe542d3b736af377562ecaa/0000181028_de.pdf?rev=ec8dfd6ea34947688ab70f343d481a7e&hash=01A6578B4726370C788A35703EE5036D">datasheet</a>. -->
+      <!-- KUKA KR 270 R2700 ultra - 0000181028_DE.pdf, version 0000-181-028 / V46.1 / 07.01.2021 / de -->
```

Comment:
Please mention the document information as regular text, not as an XML comment.

Something like [kuka_kr150_support/package.xml](https://github.com/ros-industrial/kuka_experimental/blob/e906e76700cec944c3f04728157f487b771cf399/kuka_kr150_support/package.xml#L20-L23).

Reply:
see db4e0d0608eaa9675a522993c4004d35734fadaf

I cannot include the link as non-comment, since then the package.xml cannot be parsed anymore by catkin ("no valid xml syntax")